website/common/script/content/index.js

```diff
@@ -2519,7 +2519,11 @@ api.subscriptionBlocks = {
   basic_12mo: {
     months: 12,
     price: 48
-  }
+  },
+  group_monthly: {
```

Comment:
Let's add a yearly plan anyway with same pricing as the user's yearly plan. We can change it later